forem/forem
# Refactor VerifyAndDismissFlashMessage within Cypress tests

## What type of PR is this? (check all applicable)


## Description

Recently, in this https://github.com/forem/forem/pull/19699, we'd build out functionality that mimics the member manager page. Hence, we copied some of its cypress helpers, specifically we'd created a helper called [verifyAndDismissFlashMessage](https://github.com/forem/forem/pull/19699/files/4a6a642722dd35e134873072e35f674491c8d5e0#diff-d1842c6bcb885cbb271a78f72ee042caf8f969c2c0113c4e4e338bd068a69891).

In order to clean up the code, we want to extract out the [verifyAndDismissUserUpdatedMessage](https://github.com/forem/forem/blob/rt4914/bulk-assign-roles/cypress/e2e/seededFlows/adminFlows/users/userAdminUtilitites.js) utilities and replace it with the shared utility [verifyAndDismissFlashMessage](https://github.com/forem/forem/pull/19699/files/4a6a642722dd35e134873072e35f674491c8d5e0#diff-d1842c6bcb885cbb271a78f72ee042caf8f969c2c0113c4e4e338bd068a69891)
and be able to reuse it across our cypress tests to test the verification flash message pattern. The [verifyAndDismissFlashMessage](https://github.com/forem/forem/pull/19699/files/4a6a642722dd35e134873072e35f674491c8d5e0#diff-d1842c6bcb885cbb271a78f72ee042caf8f969c2c0113c4e4e338bd068a69891) will need to be updated accordingly.

## Related Tickets & Documents

- Related Issue #19762 
- Closes #19762 

## QA Instructions, Screenshots, Recordings

None really, CI will make sure Cypress tests run without failures on related files

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [x] No, and this is why:updating test utility

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/9PtfS5tTC8ejlYfCLU/giphy.gif)
